pkg/local-gadget-manager/local-gadget-manager_test.go

```diff
@@ -593,7 +593,7 @@ func TestNetworkGraph(t *testing.T) {
 		t.Fatalf("Failed to start the tracer: %s", err)
 	}
 
-	runTestContainer(t, containerName, "docker.io/library/alpine", "wget https://1.1.1.1", "")
+	runTestContainer(t, containerName, "docker.io/library/nginx", "nginx & sleep 1; curl 127.0.0.1", "")
```

Comment:
nit: AFAIK, `nginx` command returns when it finishes to start up the server, so it could simply be `nginx; curl 127.0.0.1`.